spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtils.java

```diff
@@ -34,15 +38,26 @@
 	private Fabric8ConfigUtils() {
 	}
 
-	public static String getApplicationNamespace(KubernetesClient client, String configNamespace,
+	public static String getApplicationNamespace(KubernetesClient client, String namespace,
```

Comment:
`configNamespace` to `namespace`, to stay on par with the other method present in this class